x12-edi/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,593 @@
+= X12 EDI Connector - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium] +
+X12 EDI Connector Version 2.0.0
+
+X12 EDI Connector lets you convert X12 messages to and from DataWeave-compatible representations using lists and maps.
+The schema definitions for all https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-x12[supported X12 standard transaction sets] are provided.
+Many HIPAA document versions are also supported, as listed separately in the https://docs.mulesoft.com/connectors/x12-edi/x12-edi-versions-hipaa[Connector HIPAA reference page].
+
+== Prerequisites
+
+This document assumes that you are familiar with X12, Mule, Anypoint
+Connectors, Anypoint Studio, Mule flows, and Mule Global Elements.
+
+See the Release Notes for compatibility information. A link to the
+release notes is listed in the See Also section of this document.
+
+To use the X12 EDI connector in a production environment, you must
+have purchased a MuleSoft license for Anypoint B2B.
+
+NOTE: X12 EDI Connector supports files of up to 15 MB, and the memory requirement is roughly 40 to 1. For example, a 1-MB file requires up to 40 MB of memory to process, so it's important to consider this memory requirement in conjunction with your TPS needs for large X12 files. This is not an exact figure; the value might vary according to the complexity of the mapping instructions.
+
+== What's New in this Connector
+
+X12 EDI connector 2.x makes several changes to the configuration and handling implemented by X12 EDI Connector 1.x.
+
+* X12 EDI connector 1.x only supports working with transaction sets from a single version of the standard. Mule 4 X12 EDI connector 2.x supports working with transaction sets from multiple versions of the standards. The version information included in the group (GS segment) data is used to identify the standard version, and the transaction set identifier determines the particular schema.
+
+* Clearer configuration options are provided for standard X12 vs HIPAA usage. The formValidation parameter selects whether you're using standard X12, HIPAA with SNIP Type 1 validation, or HIPAA with SNIP Type 2 validation (other SNIP validation types are not implemented at the connector level, but can be implemented in your application code). Only standard X12 schemas may be used when X12 is selected; only HIPAA schemas may be used when either HIPAA option is selected.
```

Comment:
Universal: may > can
Suggestion: where possible, get rid of the parentheses and put the text in a second sentence.

Reply:
fixed.